### PR TITLE
Add option to clean up test ledger after simpletest.

### PR DESCRIPTION
**Pull request by MarvinCai**
### Motivation
resolves: https://github.com/apache/bookkeeper/issues/3473

### Changes
Add option to clean up test ledger after simpletest.
Also add metadata for ledger created by simpletest so we have more confidence when clean them up manually.

### CODE REVIEW

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/client/SimpleTestCommand.java [hunk 1]
```diff
@@ -96,7 +101,10 @@ protected void run(BookKeeper bk, Flags flags) throws Exception {
                     lastReport = System.nanoTime();
                 }
             }
-            LOG.info("{} entries written to ledger {}", flags.numEntries, wh.getId());
+            LOG.info("{} entries written to ledger {}, cleaning up the ledger", flags.numEntries, wh.getId());
```
Comment (dlg99) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/client/SimpleTestCommand.java:104]: nit: i'd move logging of "cleaning up the ledger" under `if (flags.cleanup)`

**Review by dlg99 — APPROVED** [review]
lgtm

**Review by hangc0276 — COMMENTED** [review]
IMO, we can clean those ledgers by default.

**Review by StevenLuMT — APPROVED** [review]
lgtm

**Review by eolivelli — APPROVED** [review]
LGTM

**Pull request merged**